pkg/eval/path/path.go

```diff
@@ -0,0 +1,94 @@
+// Package path exposes functionality from Go's path package as an Elvish
+// module.
+package path
+
+import (
+	"path/filepath"
+
+	"github.com/elves/elvish/pkg/eval"
+)
+
+//elvdoc:fn abs
```

Comment:
Most of these doc strings are not detailed enough, compared to the godoc counterparts. Either add links to the godoc, or adapt the godoc.